File: Iniciante/1066.py
contador_par = 0
contador_impar = 0
valores_positivos = 0
valores_negativos = 0
for i in range(5):
    valores = int(input())
    if valores % 2 == 0:
        contador_par += 1
    if valores % 2 != 0:
        contador_impar += 1
    if valores > 0:
        valores_positivos += 1
    if valores < 0:
        valores_negativos += 1
print('{} valor(es) par(es)'.format(contador_par))
print('{} valor(es) impar(es)'.format(contador_impar))
print('{} valor(es) positivo(s)'.format(valores_positivos))
print('{} valor(es) negativo(s)'.format(valores_negativos))

File: Iniciante/1066.cpp
#include <iostream>

using namespace std;

int main(){
    int valores;
    int lista[4] = {};
    for (int i=0; i<5;i++){
        cin >> valores;
        if (valores % 2 == 0){
            lista[0] += 1;
        }
        if (valores % 2 != 0){
            lista[1] += 1;
        }
        if (valores > 0){
            lista[2] += 1;
        }
        if (valores < 0){
            lista[3] += 1;
        }
    }
    cout << lista[0] << " valor(es) par(es)" << endl;
    cout << lista[1] << " valor(es) impar(es)" << endl;
    cout << lista[2] << " valor(es) positivo(s)" << endl;
    cout << lista[3] << " valor(es) negativo(s)" << endl;
    return 0;
}
